recipe/meta.yaml

```diff
@@ -16,13 +16,14 @@ build:
 
 requirements:
   host:
-    - python >=3.7,<3.10
+    - python >=3.10,<3.13
     - pip
   run:
-    - python >=3.7,<3.10
+    - python >=3.10,<3.13
```

Comment:
3.12 not allowed upstream https://github.com/jthorton/de-forcefields/blob/1.0.1/setup.cfg#L43-L44